tests/test_anker.py

```diff
@@ -388,3 +389,108 @@ def deposit_solido_sol(amount_sol: float) -> str:
 assert result['st_sol_reserve_balance_st_lamports'] == 1_000_000_000
 assert result['b_sol_supply_b_lamports'] == 0
 print(f'> Anker reserve has 1 stSOL, the bSOL mint has a supply of 0 bSOL.')
+
+print('\nTesting manager functions ...')
+print('> Changing Terra rewards destination')
+transaction_result = solido(
+    'anker',
+    'change-terra-rewards-destination',
+    '--anker-address',
+    anker_address,
+    '--multisig-address',
+    multisig_instance,
+    '--multisig-program-id',
+    multisig_program_id,
+    '--terra-rewards-destination',
+    'terra14dycr8jm7e5kw88g4studekkzzw5xc5ffnp4hk',
```

Comment:
Let’s put this in a variable `new_terra_rewards_destination` or something like that, so we don’t have to repeat it below. Also we can assert that it is different from the previous address, so we are sure that we are actually changing something.